tasks/main.yml

```diff
@@ -40,11 +43,14 @@
   targetcli_iscsi_acl:
     wwn: "{{ item.0.wwn }}"
     initiator_wwn: "{{ item.1 }}"
+    initiator_user: "{{ item[0]['initiators_auth'][item.1]['user'] | default('') }}"
+    initiator_password: "{{ item[0]['initiators_auth'][item.1]['password'] | default('') }}"
   with_subelements:
     - "{{ iscsi_targets }}"
     - initiators
   notify:
     - save targetcli configuration
+  #no_log: true
```

Comment:
this should not be present here but in `OndrejHome.targetcli-modules` ansible role modules